wpilibc/src/main/native/include/ErrorBase.h

```diff
@@ -56,9 +56,9 @@
                       __FUNCTION__, __LINE__)
 #define wpi_setStaticWPIError(object, error) \
   wpi_setStaticWPIErrorWithContext(object, error, "")
-#define wpi_setGlobalWPIErrorWithContext(error, context)                \
-  ::frc::ErrorBase::SetGlobalWPIError((wpi_error_s_##error), (context), \
```

Comment:
While I typically prefer not to use `::`, in macros it makes sense because you really have no idea of the context where they'll be invoked.

Reply:
Good point.

Reply:
Looks like you missed one.. there were two places you changed that in ErrorBase.h